.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: python
+
+matrix:
+    include:
+        python: 3.6
+        os: linux
+        env:
+            - CTAPIPE_VERSION=master
+            - PROTOZFITS_VERSION=v1.4.2
+            - CTAPIPE_IO_NECTARCAM_VERSION=master
+
+
+before_install:
+
+    # Use utf8 encoding. Should be default, but this is insurance
+    # against future changes
+    - export PYTHONIOENCODING=UTF8
+    - export MPLBACKEND=Agg
+
+    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - . $HOME/miniconda/etc/profile.d/conda.sh
+    - hash -r
+    - conda config --set always_yes yes --set changeps1 no
+    - conda update -q conda  # get latest conda version
+    - conda info -a # Useful for debugging any issues with conda
+
+install:
+    - conda env create --name cta --file environment.yml
+    - conda activate cta
+    - pip install https://github.com/cta-observatory/ctapipe/archive/$CTAPIPE_VERSION.tar.gz
```

Comment:
This appears to be inconsistent with the `envrionment.yml` file proposed: the conda environment will first install `ctapipe` from conda, and then again from `pip` here.

Reply:
I think that this installation is only for travis which will run the tests. Unfortunately I don't know how to test it before accepting the PR

Reply:
I think I get it: this is actually consistent with the instructions provided in the `README`, but only if we don't install `ctapipe` via `pip` via the environment file itself (see replies to comments above). If `ctapipe` is to be installed via `pip` in the environment file, then this statement in the `travis` configuration file is redundant with the instruction `conda env create --name cta --file environment.yml`.

Reply:
is there a reason you don't just do `conda install -c cta-observatory ctapipe=$CTAPIPE_VERSION` ?